src/components/Person/Person.jsx

```diff
@@ -1 +1,46 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const marrieds = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${marrieds}`
+          : 'I am not married'
+        }
+      </p>
+    </section>
+  );
+};
+
+Person.propTypes = {
```

Comment:
i think it will not working. Use `PropTypes.shape()`

Reply:
But it`s work)

Reply:
yeah, Lint didn't notice that, and i don't know why, but you can see in browser console some errors because of that